solutions/USACO jan17-balanced-photo.mdx

```diff
@@ -0,0 +1,64 @@
+---
+id: usaco-693
+source: USACO Gold 2017 January
+title: Balanced Photo
+author: Andy Wang
+---
+
+[Official Analysis](http://usaco.org/current/data/sol_bphoto_gold_jan17.html)
+
+The official solution uses [binary indexed tree](https://usaco.guide/gold/PURS/#binary-indexed-tree)
```

Comment:
```suggestion
The official solution uses a [binary indexed tree](https://usaco.guide/gold/PURS/#binary-indexed-tree).
```

Reply:
change link to `/gold/PURS/#binary-indexed-tree`.